### ISSUE TITLE
Extension activation fails with error: RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by morphx666** [swdotcom/swdc-vscode#376]
**Describe the bug**
```paste below
After upgrading to VS Code 1.63.2 the extensions fails to load with the following error (from the ExtensionHost log):

[exthost] [error] Activating extension softwaredotcom.swdc-vscode failed due to an error:
[exthost] [error] RangeError: Maximum call stack size exceeded
```
Full Log Attached: [error-log.txt](https://github.com/swdotcom/swdc-vscode/files/7747479/error-log.txt)

**ISSUE TYPE**
- Bug Report

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Plugin version: Code Time v2.6.19

**Comment by morphx666** [comment]
Now running v2.6.21, but the problem persists.
I have the exact same version of VS Code + the Code Time plugin on my laptop (running Ubuntu) and it works just fine.

**Comment by xavluiz** [comment]
We'll look into this right away, thank you for the update.

**Comment by xavluiz** [comment]
Hi @morphx666,
Is this still an issue with v2.6.22? Would you happen to have the output from the "CodeTime" channel as well? This will help in identifying what might be happening, thank you.

**Comment by morphx666** [comment]
I haven't tested it with  v2.6.22 - I only see v2.6.21 available on VS Code's marketplace.

As for the output from CodeTime, is completely empty.

**Comment by xavluiz** [comment]
You're right. That version is currently in review. We'll look to get this available by next week. Thank you.

**Comment by xavluiz** [comment]
Hi @morphx666,

v2.6.22 has been released. We've tested it on VSCode version 1.63.2 and haven't been able to reproduce the range error but let us know if it still a concern. Thank you.

**Comment by morphx666** [comment]
Unfortunately, it still doesn't work - but don't worry too much about it as it seems to be something related to this particular PC. It's working perfectly fine on three other computers.

I'll post back if I'm ever able to find a solution.

**Comment by xavluiz** [comment]
Ok, sounds good. Thank you for the update.

**Comment by morphx666** [comment]
Since today I had a couple of hours to spare I decide it to try to debug the compiled extension code.

I was able to find that the error occurs in this function:

```
t.getJsonItem = function (e, t) {
  const r = i ? i(e) : null;
  return r ? r[t] : null
}
```

I added some `console.log`s to have an idea of what was going on and it appears that, for some reason, the `i` function (which maps to `getFileDataAsJson`) was always null, even if I could see it declared just a couple of lines above.

So I added some code to log the other variables values whenever `i` was null.

Doing so, allowed the extension to load (instead of faulting because `i(e)` was null).
And... for whatever reason, this fixed the problem! I got the dialog to login, I logged in using my account and the extension is now working again.

I have no idea what I did to fix it, but hopefully this might give you some insight on what may have gone wrong.

Thanks again for such a great extension!
